src/components/LinkerPage/LinkerPage.tsx

```diff
@@ -1,42 +1,40 @@
 import React from 'react'
 import { Address, Blockie, Header, Button, Navbar } from 'decentraland-ui'
-// import Navbar from 'decentraland-dapps/dist/containers/Navbar'
 import { ChainId } from '@dcl/schemas'
-
 import { getConfig, isDevelopment } from '../../config'
 import { coordsToString } from '../../modules/land/utils'
 import Error from '../Error'
 import { Props } from './types'
 
-export default class LinkScenePage extends React.PureComponent<Props> {
-  handleSignature = (e: any) => {
+export default function LinkScenePage(props: Props) {
+  const handleSignature = (e: any) => {
     e.preventDefault()
-    const { onSignContent } = this.props
+    const { onSignContent } = props
     onSignContent(getConfig('rootCID'))
   }
 
-  getLANDname() {
-    const { base } = this.props
+  function getLANDname() {
+    const { base } = props
     return base.name ? `"${base.name}"` : `LAND without name`
   }
 
-  getFormattedUnauthorized() {
-    const { authorizations } = this.props
+  function getFormattedUnauthorized() {
+    const { authorizations } = props
     return authorizations
       .filter(a => !a.isUpdateAuthorized)
       .map(a => `"${coordsToString(a)}"`)
       .join(', ')
   }
 
-  renderWalletData() {
+  function renderWalletData() {
```

Comment:
since we are moving this to fns, and all we do is receive props and render some html, could all this fn be separate componetns ?

Reply:
this way instead of calling a fn that returns another instance all the time.
We can have pure components, <LandInfo { ..props} /> <WalletData /> etc..